contracts/strategies/interfaces/aave/IStakedAave.sol

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.6;
```

Comment:
I think this file already exists at `contracts/strategies/aave/interfaces/IStakedAave.sol`.